eng/Versions.props

```diff
@@ -137,35 +125,27 @@
     <MonoAddinsPackageVersion>1.3.8</MonoAddinsPackageVersion>
     <MonoDevelopSdkPackageVersion>1.0.15</MonoDevelopSdkPackageVersion>
     <MoqPackageVersion>4.16.0</MoqPackageVersion>
-    <!-- STOP!!! We need to reference the version of JSON that our HOSTS support. -->
-    <NewtonsoftJsonPackageVersion>12.0.2</NewtonsoftJsonPackageVersion>
```

Comment:
Seems like we just don't use Newtonsoft anymore? A welcome change, though surprising on this branch. What lead to that?